### PR TITLE
Make args translation command specific

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@meltsufin @etanshaul

**Comment by meltsufin** [comment]
There is some code duplication between the three classes where some methods are identical. Can we eliminate that somehow? Maybe Args class with three static inner classes where they delegate to methods in the parent class?

**Comment by loosebazooka** [comment]
Hrmm... interesting lemme try that.

File: src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/args/GcloudArgs.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   * @return [--name, file.getAbsolutePath()] or [] if file=null.
+   */
+  public static List<String> get(String name, File file) {
+    return Args.filePath(name, file);
+  }
+
+  /**
+   * @return [key1=value1,key2=value2,...], [] if keyValueMapping=empty/null
+   */
+  public static List<String> keyValues(Map<?, ?> keyValueMapping) {
```
Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/args/GcloudArgs.java]: Should this be in Args as well?

**Comment by loosebazooka** [comment]
rebased, PTAL

**Comment by meltsufin** [comment]
:ship: :+1:

**Pull request merged**